langchain/agents/agent_toolkits/spark/prompt.py

```diff
@@ -0,0 +1,13 @@
+# flake8: noqa
+
+PREFIX = """
+You are working with a spark dataframe in Python. The name of the dataframe is `spdf`.
```

Comment:
Nit: shall we make the dataframe name as `df`? Probably you are trying to make a difference from the panda dataframe, but many Spark users are using `df`.